internal/pkg/service/cli/cmdconfig/binder.go

```diff
@@ -14,16 +18,31 @@ const (
 type Binder struct {
 	envNaming *env.NamingConvention
 	envs      env.Provider
+	logger    log.Logger
 }
 
-func NewBinder(envs env.Provider) *Binder {
+func NewBinder(envs env.Provider, l log.Logger) *Binder {
 	return &Binder{
 		envNaming: env.NewNamingConvention(ENVPrefix),
 		envs:      envs,
+		logger:    l,
 	}
 }
 
-func (b *Binder) Bind(flags *pflag.FlagSet, args []string, targets ...any) error {
+func (b *Binder) Bind(ctx context.Context, flags *pflag.FlagSet, args []string, targets ...any) error {
 	cfg := configmap.BindConfig{Flags: flags, Args: args, EnvNaming: b.envNaming, Envs: b.envs}
-	return configmap.Bind(cfg, targets...)
+	err := configmap.Bind(cfg, targets...)
+	if err != nil {
+		return err
+	}
+
+	for _, v := range targets {
+		dump, err := configmap.NewDumper().Dump(v).AsJSON(false)
+		if err == nil {
+			b.logger.Debugf(ctx, "Global flags: %s %s", reflect.ValueOf(v).Type().String(), string(dump))
```

Comment:
Is there a typo in message prefix? `Global flags` ? Because this method is used also by command flags.

So if you don't like suggested `Binded flags:`, you can use `Flags:`.

Reply:
yes, It's typo.

Reply:
fixed: fda87a711c7ef34467f24066f24630f12ad234c7